ci-operator/step-registry/rosa/sts/account-roles/create/rosa-sts-account-roles-create-commands.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+ACCOUNT_ROLES_PREFIX=${ACCOUNT_ROLES_PREFIX:-"ManagedOpenshift"}
```

Comment:
Can we use job hash to decide the prefix, but not provided by users.

Reply:
The `ManagedOpenshift` is the default prefix on rosa cli side. For me, I'd like the user can define what ACCOUNT_ROLES_PREFIX as they like but not a random string. The account-role can be used by multiple clusters, and no conflict during creating/using.

Reply:
Think again, to avoid the account-roles are deleted by others. Changed to `ACCOUNT_ROLES_PREFIX=${ACCOUNT_ROLES_PREFIX:-$NAMESPACE}`.